ui/src/app/features/user-table/user-table.component.ts

```diff
@@ -39,6 +41,12 @@ export class UserTableComponent extends TableComponent implements OnInit, OnChan
   @Input() searchText: string;
   @Output() deleteUser = new EventEmitter<UserDeletion>();
 
+  constructor(private matIconRegistry: MatIconRegistry, private domSanitzer: DomSanitizer) {
+    super();
+    this.matIconRegistry.addSvgIcon('edit', this.domSanitzer.bypassSecurityTrustResourceUrl('assets/img/icons/edit.svg'));
```

Comment:
same question as for add SVG icon. I can see that some icons are not registered following this pattern. Do we need it? if yes can be done in 1 place?

Reply:
other icons are not using for now